Automattic/wp-calypso
# Help Center: Add site type check on help center

#### Proposed Changes
Add a site type check on help-center.
If site is atomic it should never perform a request to `https://public-api.wordpress.com/wpcom/v2/sites/208767849/block-editor/has-seen-whats-new-modal?_envelope=1` endpoint.

No requests on atomic sites
<img width="1187" alt="image" src="https://user-images.githubusercontent.com/2653810/181285759-520f18b2-041f-4c07-a43b-6a1a1ac181a2.png">

Requests on simple sites
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/2653810/181285879-e2f261d4-7e5b-4df0-bb6d-d5540204556b.png">


#### Testing Instructions
* Pull this branch `fix/add-site-type-check-help-center`
* Run `yarn start`
* Navigate to your site, for instance `http://calypso.localhost:3000/post/yoursite.com`
* Open the network tab and check for api requests to `wpcom/v2/sites/208767849/block-editor/has-seen-whats-new-modal`

Check: If your site is a simple site, you should see the requests. If atomic you should not

Related to #65974
Fixes #65974
